easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -104,8 +133,27 @@ def configure_step(self, srcdir=None, builddir=None):
 
         options = ['-DCMAKE_INSTALL_PREFIX=%s' % self.installdir]
 
-        if self.cfg['build_type'] is not None:
-            options.append("-DCMAKE_BUILD_TYPE=%s" % self.cfg['build_type'])
+        if '-DCMAKE_BUILD_TYPE=' in self.cfg['configopts']:
+            if self.cfg['build_type'] is not None:
+                self.log.warning('CMAKE_BUILD_TYPE is set in configopts. Ignoring build_type')
```

Comment:
should be `print_warning` here too?

Reply:
There is no way to silence this warning e.g. if it is intended to build release and debug unless build_type can be a list like configopts and EB iterators over it. Hence "only" a log warning

Reply:
OK